client/src/components/Public/Map/Map.js

```diff
@@ -82,10 +132,26 @@ export default compose(
     clusters: clustersSelector(state),
     addressesUrl: addressesUrlSelector(state),
   })),
-  withDataProviders(({addressesUrl}) => [addressesProvider(addressesUrl)]),
+  withProps((props) => {
+    return {
+      useLabels: !(props.zoom >= CITY_ZOOM ||
+  !isInSlovakia(props.center)),
```

Comment:
This condition is really hard to grasp (probably because of the double negative). Could you rewrite it by DeMorgan :slightly_smiling_face: ?

Reply:
done